mongodbatlas/data_lakes.go

```diff
@@ -0,0 +1,236 @@
+package mongodbatlas
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+)
+
+const (
+	dataLakesBasePath = "groups"
+)
+
+// DataLakeService is an interface for interfacing with the Data Lake endpoints of the MongoDB Atlas API.
+// See more: https://docs.mongodb.com/datalake/reference/api/datalakes-api
+type DataLakeService interface {
+	List(context.Context, *DataLakeReqPathParameters, *ListOptions) ([]DataLake, *Response, error)
```

Comment:
reading the docs it doesn't look like we support `ListOptions` for this endpoint